### ISSUE TITLE
Reproduction of the published result

### ISSUE THREAD

**Issue by hongsukchoi** [nkolot/SPIN#96]
Hi, thanks for your great work.

I want to reproduce the numbers of SPIN's performance on Human3.6M, which are written on the [published paper](https://arxiv.org/abs/1909.12828). 
But unfortunately, this repository does not contain proper preprocessing step for Human3.6M test set (Subject 9,11).
![스크린샷 2020-11-09 09 47 56](https://user-images.githubusercontent.com/31090011/98489300-bb5c4500-2270-11eb-9e71-6358edeb954d.png)
(where can I get those 'My~' data?)

Thus, I used preprocessed Human3.6M test data from [here](https://github.com/mks0601/I2L-MeshNet_RELEASE).
And, I evaluated the model with your pre-trained weights.
Following the standard protocol, I sub-sampled every 64th frame, and tested on the frontal camera.
Below is the result.

![스크린샷 2020-11-09 09 53 10](https://user-images.githubusercontent.com/31090011/98489595-00cd4200-2272-11eb-96f5-93aba334c3bf.png)

Clearly, PA-MPJPE increase by 3mm which is a quite big difference even considering the different preprocessing.
Could you explain why this happens?
And I hope you can provide proper evaluation steps including the preprocessing for the reproduction.
If I am missing something, plz let me know.

Thank you!

**Comment by Youwang-Kim** [comment]
안녕하세요 최홍석님. 제가 알아낸 바로는, 이런 식으로 저장된 것 같습니다. 해당 분야 연구를 이제 막 관심있게 들여다보는 석사 1학기차 포스텍 학생인데, SMPL 관련해서 human3.6M 데이터셋이 거의 모든 repository에서 문제가 되는 것 같아 곤란하네요.. ㅎㅎ

S9 -|-- Videos
       |--MySegmensMat
       |       \--ground_truth_bb : human3.6 training data - subject S9 - Segments - BBoxes MAT 
       |                                           (Human3.6M 홈페이지에서 다운받을 수 있는 경로를 적어둔 것입니다)
       |       \--ground_truth_bs : human3.6 training data - subject S9 - Segments - BackSub MAT
       | --MyPoseFeatures
              \-- D2_Positions_mono : human3.6 training data - subject S9 - Poses - D2 Positions mono
              \-- D3_Positions_mono : human3.6 training data - subject S9 - Poses - D3 Positions mono


혹시 다른 방식으로 preprocessed Human3.6M 데이터를 사용하신 것에 대해서 디테일하게 설명해주실 수 있나요? 
저도 그 방법으로 h3.6m_train.npz를 사용해보고 싶습니다.
언급하신 I2L-MeshNet 논문의 방법으로 human3.6M 데이터에 대한 SMPL parameter를 뽑아내신건가요?

**Comment by hongsukchoi** [comment]
안녕하세요 데이터셋 구조에 대해 알려주셔서 감사합니다! 혹시 이 레포에 나온대로 preprocessing해 여기서 제공된 pretrained된 model로 evaluate했을 때 논문에 나온 숫자가 재현되던가요? 저도 알려주신 구조로 preprocessing을 한 후 evaluation해보려합니다.

I2L-MeshNet이나 Pose2Mesh에서 사용한 SMPL parameter는 SMPLify-X라는 최신 optimization 방법으로 얻었습니다.
자세한 내용은 두 논문의 supplement에서 찾아보실 수 있습니다.

**Comment by Youwang-Kim** [comment]
이 레포에 나온대로 preprocessing 하려고 했는데, preprocess_datasets.py 실행 시 spacepy library import 관련해서 문제가 조금 생겨서 아직 해결 중에 있습니다..! 해결한다면 다시 알려드리도록 할게요.

**Comment by Youwang-Kim** [comment]
데이터셋은 저도 보니까 D3 Positions mono 대신 D3 Positions 로 들어가있어서, 이거 수정해서 메일로 연락드릴게요..!

**Comment by hongsukchoi** [comment]
저도 spacepy library 관련해 문제가 있었는데, 제 경우에는 https://spacepy.github.io/install_linux.html 여기 참고해서 CDF를 직접 다운받고 install하니까 해결됐습니다. 감사합니다!

**Comment by Youwang-Kim** [comment]
혹시 cdf 다운 받는 과정에서 nasa.gov 홈페이지 access가 되던가요? 
저도 저 방법으로 하려고 했는데, cdf 다운로드하는 링크 액세스가 안돼서 다른 라이브러리 찾아서 우회해서 하는 중입니다만..
